cmake/projects/tomcrypt/hunter.cmake

```diff
@@ -41,6 +41,16 @@ hunter_add_version(
     SHA1
     69b882eff5bf69a5a4e0e9169cc9b9c0ea5b9d71
 )
+hunter_add_version(
+    PACKAGE_NAME
+    tomcrypt
+    VERSION
+    "1.18.2-p0"
+    URL
+    "https://github.com/hunter-packages/libtomcrypt/archive/v1.18.2-p0.tar.gz"
+    SHA1
+    8acf74d6a3af6c4360e665a5df0ba84ca3568d0e
```

Comment:
SHA1 value is not correct. Note that you will discover this fact if you will run tests properly.